skada/tests/test_mapping.py

```diff
@@ -171,23 +192,33 @@ def _base_test_new_X_adapt(estimator, da_dataset):
         (LinearOTMappingAdapter()),
         (CORALAdapter()),
         (CORALAdapter()),
-        (
-            pytest.param(
-                MMDLSConSMappingAdapter(gamma=1e-3),
-                marks=pytest.mark.skipif(not torch, reason="PyTorch not installed"),
-            )
-        ),
-        (
-            pytest.param(
-                MMDLSConSMappingAdapter(gamma=1e-3),
-                marks=pytest.mark.skipif(not torch, reason="PyTorch not installed"),
-            )
-        ),
+        # (
```

Comment:
commented out for debugging? should be brought back no?

Reply:
good catch, done